client/src/components/Health/HealthCard.jsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
```

Comment:
@FikriAnuar this could be simplified to import { Button, Card, CardActions, CardContent, CardMedia, Typography } from '@mui/material'